### PR TITLE
Add first docs for .NET MAUI

### PR DESCRIPTION
**Pull request by mattjohnsonpint**
Adding initial documentation for .NET MAUI

### CODE REVIEW

File: src/platforms/dotnet/guides/maui/index.mdx
Comment (imatwawana) [inline src/platforms/dotnet/guides/maui/index.mdx:45]: We don't use exclamation points in docs and we want to avoid language beyond appreciation.
```suggestion
The `Sentry.Maui` SDK is currently in public preview.  All APIs added by this package are subject to breaking changes.  We appreciate your feedback:
```

File: src/platforms/dotnet/guides/maui/index.mdx
Comment (imatwawana) [inline src/platforms/dotnet/guides/maui/index.mdx:51]: Improvement is subjective, so we want to avoid saying that, but we can say objectively that the documentation will be updated

```suggestion
This documentation will be updated as we get closer to general availability release. 
```

**Review by imatwawana — COMMENTED** [review]
Made some style fixes

**Review by bruno-garcia — APPROVED** [review]
NIce! Lets merge this once we merge the release registry PR and push to NuGet

**Comment by mattjohnsonpint** [comment]
Setting to draft temporarily, just so we don't merge this until we've pushed the release and release-registry out.

**Pull request merged**
